### PR TITLE
TLDetector: process images from real camera

### PR DESCRIPTION
**Pull request by kradio3**
* Switch model weights depending on ```is_site``` flag
* Validation accuracy for real camera detection: 0.9969

### CODE REVIEW

**Review by elias3 — COMMENTED** [review]
are we using:
focal_length_x
focal_length_y
From the real camera settings (see /ros/src/tl_detector/site_traffic_light_config.yaml)

File: ros/src/tl_detector/light_classification/tl_classifier.py [hunk 1]
```diff
@@ -8,14 +8,23 @@
...
+SIM_CAM_WEIGHTS =  'sc.09860.h5'
+MODEL_PICTURE_SIZE = (224, 224)
 
 class TLClassifier(object):
-    def __init__(self):
+    def __init__(self, is_site=False):
         base_path = os.path.dirname(os.path.abspath(__file__))
-        model_file = os.path.join(base_path, 'base_rnn50.h5')
+
+        sim_camera_weights = os.path.join(base_path, SIM_CAM_WEIGHTS)
```
Comment (elias3) [inline ros/src/tl_detector/light_classification/tl_classifier.py]: Those can be incorporated in the if

**Comment by nathansherrit** [comment]
Do we have a way to test this without being able to access the car?

**Review by nathansherrit — DISMISSED** [review]
Tested in simulation and car is able to stop at red lights and continue moving again when they are green. Unable to test detection when on_site flag is set to true

**Review by elias3 — APPROVED** [review]

**Pull request merged**